### DISCUSSION TITLE
i want debug the write logic in goland, but not find  any command in main, how to debug the it

### DISCUSSION THREAD

**Discussion by wenjin227** [juicedata/juicefs#429] [Q&A] [answered]
i want debug the write logic in goland, but not find  any command in main, how to debug the it

**Comment by suzaku** [comment, marked as answer]
It's easy to debug a Go program with GoLand. Please refer to the document [here](https://blog.jetbrains.com/go/2019/02/06/debugging-with-goland-getting-started/#debugging-an-application).

What you need to do to debug a command like `./juicefs mount localhost ~/jfs` is to modify the run configuration and set the program arguments to `mount localhost /path/to/jfs`. **Note that** you can't use `~` in your IDE so you have to spell out the absolute path.

<img width="425" alt="Screen Shot 2021-05-21 at 4 04 49 PM" src="https://user-images.githubusercontent.com/116473/119103935-48692d80-ba4e-11eb-9535-c02f444ffb4a.png">
<img width="604" alt="Screen Shot 2021-05-21 at 4 06 09 PM" src="https://user-images.githubusercontent.com/116473/119104117-78183580-ba4e-11eb-9de9-5d4158e2787a.png">

**Reply by wenjin227** [reply]
ths
